sonar-project.properties

```diff
@@ -1,6 +1,6 @@
 sonar.projectKey=fh-stats
 sonar.projectName=fh-stats-nightly_master
-sonar.projectVersion=0.1.0
+sonar.projectVersion=0.1.1
```

Comment:
And here?

Reply:
Not sure about this one, both modules statsd and fh-top have their own sonar-project.properties